packages/react/src/components/DatePicker/plugins/fixEventsPlugin.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { match, keys } from '../../../internal/keyboard';
+
+/**
+ * @param {object} config Plugin configuration.
+ * @returns {Plugin} A Flatpickr plugin to fix Flatpickr's behavior of certain events.
+ */
+export default config => fp => {
+  /**
+   * Handles `keydown` event.
+   */
+  const handleKeydown = event => {
```

Comment:
currently it seems that if a user types in the first date and then uses the mouse to select the second date, https://github.com/flatpickr/flatpickr/issues/1942 also occurs. should we apply the fix to mouse inputs as well? it may be unrelated or may belong in a separate PR but it's just something I noticed during testing

Reply:
Good finding @emyarod! Given we need to change the behavior of different event handler, we'll need a separate work to address this.

Reply:
Update: A workaround for the issue here is hitting Enter key before selecting the date by mouse. (The original issue (https://github.com/flatpickr/flatpickr/issues/1942) is a desire to make Enter key pick up both start/end dates user has typed)